examples/oft-solana/README.md

```diff
@@ -263,6 +269,8 @@ With a squads multisig, you can simply append the `--multisigKey` flag to the en
 
 ### Mint OFT on Solana
 
+This is only relevant for **OFT** and **OFT Mint-And-Burn Adapter** deployments. If you opted to include the `--amount` flag in the create step, that means you already have minted some Solana OFT and you can skip this section.
```

Comment:
I think --amount doesn't work for MABA looking at createOFT script and !MABA ifs.